### ISSUE TITLE
Sanity check: Classification after Self-Supervised training

### ISSUE THREAD

**Issue by DAVEISHAN** [TengdaHan/DPC#18]
Hi Tengda, Thank you, for sharing the code and giving detailed supplementary material for this amazing work! 

I am also working on a similar self-supervised task, I see my self-supervised task loss is converging, however, it is severely hurting the supervised classification performance in the earlier epochs (~20 epochs) and still performing worse by 5-6% than the supervised baseline at the same epoch. Does this mean my self-supervised task is not good or shall I wait to converge?

Let me know about your experience with UCF101 classification, how training loss and validation accuracy differs from the baseline at initial epochs? Also, please, tell me what performance improvement you get after removing shortcuts in your self-supervised task, is shortcut removal is super-mandatory or just a cherry on the cake?

Hope to hear from you soon. 

-Ishan

**Comment by TengdaHan** [comment]
It seems you're NOT using this DPC repo, and want some suggestions on your own project.

* I don't have advice on your first question, since I don't know your self-supervised task.
* For removing shortcuts, I think it's super-mandatory for self-supervised task. Network can 'cheat' in many ways you have to find it out and let the network using some 'high-level' feature to solve the self-supervised task. 

If you have more questions, just email me. It's not an 'issue' of this repo.

**Comment by YuqiHUO** [comment]
Hi, can I ask what is the 'shortcut' means in your context? Thanks!

**Comment by TengdaHan** [comment]
When we design a self-supervised task, usually we want the network to learn 'good features'. 
But sometimes the network learns to solve the task by finding some very easy solutions, getting bad features that are not desirable -- this is what I mean by 'shortcut'.
E.g. in [Doersch et al.](https://arxiv.org/abs/1505.05192), chromatic aberration is a shortcut, that is not good for representation learning.
